packages/workspace/src/schematics/init/init.ts

```diff
@@ -329,28 +329,64 @@ function setUpCompilerOptions(
   return tsconfig;
 }
 
+function renameSync(tree: Tree, from: string, to: string, cb: (err: string) => void) {
+  if (!tree.exists(from)) {
+    cb(`Path: ${from} does not exist`);
+  } else if (tree.exists(to)) {
+    cb(`Path: ${to} already exists`);
+  } else {
+    tree.rename(from, to);
+    cb(null);
+  }
+}
+
+function renameDirSync(tree: Tree, from: string, to: string, cb: (err: string) => void) {
+  let dir: any;
+  try {
+    dir = tree.getDir(from);
+  } catch (e) {
+    cb(`Path: ${from} does not exist`);
+    return;
+  }
+  dir.visit(path => {
+    const destination = path.replace(from, to);
+    tree.rename(path, destination);
+  });
+  cb(null);
+}
+
+function updateJsonFile(tree: Tree, path: string, callback: (a: any) => any) {
```

Comment:
We have this method already :) It's named `updateJsonInTree()` in `/packages/workspace/src/utils/ast-utils.ts`

Reply:
Using it now, thanks!